.gitmodules

```diff
@@ -34,10 +34,6 @@
 	path = vendor/golang.org/x/sys
 	url = https://go.googlesource.com/sys
 	branch = 62bee037599929a6e9146f29d10dd5208c43507d
-[submodule "vendor/github.com/luci/go-render"]
```

Comment:
I think we want to keep this. What was the rationale for removing this entirely?